.github/workflows/update.yml

```diff
@@ -0,0 +1,24 @@
+name: Update with the latest colorschemes
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 0" # https://crontab.guru/every-week
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch the repository code
+        uses: actions/checkout@v3
```

Comment:
Minor note that the latest is actions/checkout@v4. Not sure if there is a massive difference or compelling reason to use the newest version